README.md

```diff
@@ -15,4 +15,46 @@ The first parameter <PATH OF "auth.whisk.system"> is mandatory and the second pa
 <OPENWHISK_HOME> will be set to the parent directory of the openwhisk-catalog if it is not provided.
 If openwhisk-catalog directory is put in the openwhisk home directory, we are able to omit the
 second parameter. Otherwise, we need to specify the correct OPENWHISK_HOME either as the second
-parameter or the environment variable to make it run.
\ No newline at end of file
+parameter or the environment variable to make it run.
+
+# About the relationship of OpenWhisk and OpenWhisk-Catalog
+
+All the packages under OpenWhisk-Catalog originates from the catalog directory under OpenWhisk. They contain
```

Comment:
We do not need to say anything about where "catalog" files came from.  Simply start by stating the purpose of catalog repo. For example, start with "The openwhisk-catalog repository contains default system and utility actions to be installed in a running OpenWhisk platform".